src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -351,7 +351,7 @@ public int ConvertDocument(XDocument document)
 
             this.XRoot = document.Root;
 
-            // Start converting the nodes at the top.
+            // Start converting the remainingNodes at the top.
```

Comment:
Are all these `nodes` to `remainingNodes` changes in comments an overzealous "Find/Replace All"?

Reply:
You called it! Fixed.